src/Monolog/Handler/HipChatHandler.php

```diff
@@ -93,6 +106,22 @@ public function __construct($token, $room, $name = 'Monolog', $notify = false, $
         $this->room = $room;
         $this->format = $format;
         $this->host = $host;
+
+        $this->setVersion($version);
+    }
+
+    /**
+     * Convenience method for setting the API version. Valid values are
+     * - HipChatHandler::API_V1
+     * - HipChatHandler::API_V2
+     * @param $version string the API version to use
+     */
+    public function setVersion($version) {
```

Comment:
Yeah the idea was that you could create a new handler without having to specify all the default args.  Having a constructor or a factory that would take arguments might be better but I didn't want to break BC by doing the constructor.  Not sure if a factory is worth it.